app/delivery/meteor_routes.js

```diff
@@ -6,7 +6,9 @@ const meteorRouter = express.Router();
 
 meteorRouter.get('/meteors', async (req, res, next) => {
     try {
-        const meteors = await getMeteors();
+        const {date, count, wereDangerousMeteors} = req.query;
+
+        const meteors = await getMeteors(date, count, wereDangerousMeteors);
```

Comment:
count and wereDangerousMeteors - boolean types but come as strings from request
please convert types

Reply:
fixed